src/main/java/de/unistuttgart/crosswordbackend/crosswordchecker/CrosswordGenerator.java

```diff
@@ -57,7 +57,7 @@ private Crossword createCrossword(Set<Question> questions) {
       .parallelStream()
       .map(Question::getAnswer)
       .map(String::toUpperCase)
-      .collect(Collectors.toCollection(ArrayList::new));
+      .collect(Collectors.toList());
```

Comment:
As I just read the docs on that: Apparently the previous approach was better as `toList` does not ensure that the list is mutable.